pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/ufoer/Etteroppgjoer.kt

```diff
@@ -0,0 +1,231 @@
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Hvis du mener at beregningene i vedlegg «Beregning av etteroppgjøret» er feil, må du melde fra til oss innen 3 uker fra du fikk dette brevet. Du vil da få en ny vurdering og et nytt vedtak.",
+                Nynorsk to "Dersom du meiner at utrekningane i vedlegg «Beregning av etteroppgjøret» er feil, må du melde frå til oss innan 3 veker frå du fekk dette brevet. Du vil då få ei ny vurdering og eit nytt vedtak.",
+                English to "If you believe that the calculations in appendix «Calculation of settlement» are incorrect, you must notify us within 3 weeks of the date you received this letter. In such case you will receive a new assessment and a new decision."
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Klagefristen på vedtaket er 6 uker etter at saken er sendt til Skatteetaten. Du finner skjema og informasjon om hvordan du klager på nav.no/klage. Du må som hovedregel begynne å betale tilbake selv om du klager på vedtaket, se forvaltningsloven § 42.",
+                Nynorsk to "Klagefristen på vedtaket er 6 veker etter at saka er send til Skatteetaten. Du finn skjema og informasjon om korleis du klagar, på nav.no/klage. Du må som hovudregel byrje å betale tilbake sjølv om du klagar på vedtaket, jf. forvaltingslova § 42.",
+                English to "The deadline for submitting an appeal against the decision is 6 weeks after the case has been forwarded to the Tax Office. You will find a form and information about how to appeal at: nav.no/klage. As a general rule you must begin to repay any amounts owed even if you appeal against the decision, see Public Administration Act Section 42."
+            )
+        }
+    }
+}
+
+object HarDuSpoersmaal : OutlinePhrase<LangBokmalNynorskEnglish>() {
+    override fun OutlineOnlyScope<LangBokmalNynorskEnglish, Unit>.template() {
+        title1 {
+            text(
+                Bokmal to "Har du spørsmål?",
+                Nynorsk to "Har du spørsmål?",
+                English to "Do you have any questions?"
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Du finner mer informasjon på nav.no/etteroppgjor. På nav.no/kontakt kan du chatte eller skrive til oss. Hvis du ikke finner svar på nav.no, kan du ringe oss på telefon 55 55 33 33, hverdager kl. 09.00 - 15.00.",
+                Nynorsk to "Du finn meir informasjon på nav.no/etteroppgjor. Du kan chatte med eller skrive til oss på nav.no/kontakt. Dersom du ikkje finn svar på nav.no, kan du ringje oss på telefon 55 55 33 33, kvardagar kl. 09:00 til 15:00.",
+                English to "You can find further information at: nav.no/etteroppgjor. At nav.no/kontakt you can chat or write to us. If you do not find answers at nav.no, you can call us on +47 55 55 33 33, weekdays 09.00 - 15.00."
```

Comment:
Telefonnummer og åpningstider bør også være i Constants.

Reply:
Denne må fremdeles fikses

Reply:
Lagt inn constants for "NAV_KONTAKTSENTER_AAPNINGSTID" og "NAV_KONTAKTSENTER_TELEFON"